aws_etl_tools/redshift_ingest/ingestors.py

```diff
@@ -12,9 +12,10 @@
 
 
 class BasicUpsert:
-    def __init__(self, file_path, destination):
+    def __init__(self, file_path, destination, manifest=False):
```

Comment:
it's not totally obvious to me that `manifest` isn't actually the manifest json or a path to a manifest in s3. what are your thoughts on renaming this to something that makes it obviously a boolean? maybe `with_manifest` or `is_manifest`?